go-controller/pkg/cni/helper_linux.go

```diff
@@ -444,23 +444,13 @@ func (pr *PodRequest) UnconfigureInterface(ifInfo *PodInterfaceInfo) error {
 		return nil
 	}
 
-	// host side interface deletion
+	// host side deletion of OVS port and kernel interface
 	ifName := pr.SandboxID[:15]
-	out, err := ovsExec("del-port", "br-int", ifName)
-	if err != nil && !strings.Contains(out, "no port named") {
-		// DEL should be idempotent; don't return an error just log it
-		klog.Warningf("Failed to delete OVS port %s %s from br-int: %v\n %q", ifName, podDesc, err, string(out))
-	}
-	err = clearPodBandwidth(pr.SandboxID)
-	if err != nil {
+	pr.deletePorts(ifName)
+
+	if err := clearPodBandwidth(pr.SandboxID); err != nil {
 		klog.Warningf("Failed to clearPodBandwidth sandbox %v %s: %v", pr.SandboxID, podDesc, err)
 	}
-	// In SmartNic (CX5) case, there is no point in deleting the representor port
-	if pr.CNIConf.DeviceID == "" {
-		if err = util.LinkDelete(ifName); err != nil {
-			klog.Warningf("Failed to delete interface %s %s: %v", ifName, podDesc, err)
```

Comment:
not sure if this is a concern, but it seems that `func (pr *PodRequest) deletePorts(ifaceName string) {` does not log any errors if `_ = util.LinkDelete(ifaceName)` fails. 
Would it be possible to [add that there](https://github.com/ovn-org/ovn-kubernetes/blob/eca984ac2cebae3b170c4978405b6808908f3c2c/go-controller/pkg/cni/helper_linux.go#L503): 
```
klog.Warningf("Failed to delete interface %s %s: %v", ifName, podDesc, err)
```

Reply:
@flavio-fernandes done